src/styles/blocks.scss

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: $body-font;
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  padding: 32px 16px 16px;
+  background-color: $card-color;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__text {
+    display: flex;
+    flex-direction: column;
+    width: 166px;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    font-family: $body-font;
+    font-style: normal;
+    margin: 0;
+    color: $secondary-accent;
+  }
+
+  &__rank {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
```

Comment:
Is the height needed? It doesn't do anything. Stars have their own heights. Container height should be determined by it's contents.